foxtrot-common/src/main/java/com/flipkart/foxtrot/common/count/CountRequest.java

```diff
@@ -46,13 +46,26 @@ public void setDistinct(boolean isDistinct) {
         this.isDistinct = isDistinct;
     }
 
+
```

Comment:
Shouldn't the validation method be done in the action, rather than here? That way, it will keep the application logic separate and inside the corresponding action. As a matter of fact, it can be added as an abstract function inside the Action class with RequestType as parameter.. no?